src/main/resources/build.properties

```diff
@@ -6,3 +6,4 @@ springerNatureAPIKey=${springerNatureAPIKey}
 astrophysicsDataSystemAPIKey=${astrophysicsDataSystemAPIKey}
 ieeeAPIKey=${ieeeAPIKey}
 biodiversityHeritageApiKey=${biodiversityHeritageApiKey}
+Select\ library=Create property
```

Comment:
this looks like it came accidentally here. Pleaes remove

Reply:
Thank you, I have removed this line.